### DISCUSSION TITLE
Support next.config.ts

### DISCUSSION THREAD

**Discussion by helbing** [vercel/next.js#50121] [Ideas]
### Goals

Can load config from `next.config.ts`

### Background

From #35969

### Proposal

I'm interested in contributing, I will use Nodejs built-in require and [unjs/jiti](https://github.com/unjs/jiti) to load config, instead of Nodejs built-in require/import.

**Comment by himyjan** [comment]
#47774
add support `postcss.config.ts` too

#50126
not sure related or not 
https://github.com/vercel/next.js/blob/canary/packages/next/src/lib/find-config.ts#L9-L27
https://github.com/vercel/next.js/blob/efcec4c1e303848a5293cef6961be8f73fd5160b/packages/next/src/lib/find-config.ts#L9-L27

**Comment by devjiwonchoi** [comment]
cc @helbing 

This [feature was shipped](https://github.com/vercel/next.js/pull/63051) on `v15.0.0-canary.60`, please give it a try and feel free to ping me if any issues found!

```
npm i next@canary
```
